.github/render-action/generate.sh

```diff
@@ -69,7 +72,6 @@ function convert_asciidoc_to_html {
 
   asciidoctor \
       -b html \
-      -a copyright="Creative Commons Attribution 4.0 International License (CC-BY-4.0)" \
```

Comment:
We already have these properties set, so the information is already present in the file that's converted here.